### PR TITLE
Create qol.md

### PR DESCRIPTION
**Pull request by stef-the**

### CODE REVIEW

**Comment by TymanWasTaken** [comment]
Why the []'s?

**Comment by stef-the** [comment]
> Why the []'s?

makes it a clickable link

**Comment by stef-the** [comment]
should do it on all the other ones actually

**Comment by TymanWasTaken** [comment]
It just shows as [link]?

**Comment by stef-the** [comment]
yes
It makes it so the link becomes hyperlink (click to website) - without it the link would just be text

**Comment by TymanWasTaken** [comment]
If you wanted it to be like [this](https://google.com) it's `[text](url)`

File: files/guides/qol.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Credits
+
+#### Source
```
Comment (TymanWasTaken) [inline files/guides/qol.md:3]: You should probably make these links look nice by using this syntax: `[text](url "hover text")`

File: files/guides/qol.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Sound Subtitles
+
+> /subtitle_config
+
+Setup your sound subtitles
+
+## Popup Events
+
+Controls menu to bind keys
```
Comment (TymanWasTaken) [inline files/guides/qol.md:24]: You should probably specify "Go to the minecraft controls menu"

File: files/guides/qol.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+https://sk1er.club/mods/popup_events
+
+https://sk1er.club/mods/tnttime
+
+
+#### Discord
+https://discord.gg/sk1er
+
+# QOL Package - How do I use it?
```
Comment (TymanWasTaken) [inline files/guides/qol.md:14]: Also add something say what these mods are / what they do

**Review by TymanWasTaken — CHANGES_REQUESTED** [review]

**Pull request merged**